tests/ert_tests/shared/test_libres_facade_summarydata.py

```diff
@@ -0,0 +1,82 @@
+import logging
+from unittest import TestCase
+from unittest.mock import patch
+import pytest
+
+from utils import SOURCE_DIR
+from ert_utils import tmpdir
+import pandas as pd
+
+from ert_shared.libres_facade import LibresFacade
+from res.enkf import EnKFMain, ResConfig
+
+from res.enkf.export import SummaryCollector
+
+_orig_loader = SummaryCollector.loadAllSummaryData
+
+
+def _add_duplicate_row(*args, **kwargs):
+    df = _orig_loader(*args, **kwargs)
+    # Append copy of last date to each realization
+    idx = pd.MultiIndex.from_tuples(
+        [(i, df.loc[i].iloc[-1].name) for i in df.index.levels[0]]
+    )
+    df_new = pd.DataFrame(0, index=idx, columns=df.columns)
+    return pd.concat([df, df_new]).sort_index()
+
+
+class LibresFacadeSummarydataTest(TestCase):
+    # TODO: same as in test_libres_facade.py - perhaps move into utils?
+    def facade(self):
+        config_file = "snake_oil.ert"
+
+        rc = ResConfig(user_config_file=config_file)
+        rc.convertToCReference(None)
+        ert = EnKFMain(rc)
+        facade = LibresFacade(ert)
+        return facade
+
+    @pytest.fixture(autouse=True)
+    def inject_fixtures(self, caplog):
+        self._caplog = caplog
```

Comment:
This is a bit strange to me, the way you are using caplog. Why not just use it for the tests that require it? 
as an example check out `test_plugin_manager.py:test_workflows_merge_duplicate`

Reply:
I added explanations in code